cas/ut/client_ut.c

```diff
@@ -2063,8 +2063,8 @@ static void next_bulk(void)
 	vals_create(COUNT, COUNT_VAL_BYTES, &keys);
```

Comment:
How about renaming `vals_create()` to `vals_create_aligned()`?

The patch looks fine. If we can rename this or add comments for `vals_create()`, it would be better.

```
diff --git a/cas/ut/client_ut.c b/cas/ut/client_ut.c
index 3194c4c3..c204340a 100644
--- a/cas/ut/client_ut.c
+++ b/cas/ut/client_ut.c
@@ -140,6 +140,10 @@ static void value_create(int size, int num, char *buf)
        }
 }

+/**
+ * Allocates bufvec(s) aligned.
+ * The result must be freed by m0_bufvec_free_aligned().
+ */
 static void vals_create(int count, int size, struct m0_bufvec *vals)
 {
        int i;

```

Reply:
@nikitadanilov : Can you reply to the comment from @huanghua78  ?